lib/galaxy/schema/__init__.py

```diff
@@ -1,13 +1,44 @@
-import typing
+from typing import (
+    Dict,
+    List,
+    Optional,
+)
 
 from pydantic import BaseModel
 
 
 class BootstrapAdminUser(BaseModel):
     id = 0
-    email: typing.Optional[str] = None
-    preferences: typing.Dict[str, str] = {}
+    email: Optional[str] = None
+    preferences: Dict[str, str] = {}
     bootstrap_admin_user = True
 
     def all_roles(*args) -> list:
         return []
+
+
+class Tour(BaseModel):
+    id: str
+    name: str
+    description: str
+    tags: List[str]
```

Comment:
Will the documentation of the fields (for the OpenAPI docs) be added in a separate PR?

Reply:
@davelopez I can add it now. What are the required fields? (I'm looking at OpenAPI specs..) Is it just title, description, example?

Reply:
Yes, those seem to be the most used for documentation, you can also define some restrictions about the fields with the other properties (`ge`, `max_length`, etc.) if it is required.

Reply:
Done.